tests/wpt/run.py

```diff
@@ -74,6 +78,18 @@ def set_defaults(kwargs):
     if kwargs["processes"] is None:
         kwargs["processes"] = multiprocessing.cpu_count()
 
+    if kwargs["ca_cert_path"] is None:
```

Comment:
Maybe we'll want to check `"ca-cert-path" in paths` like the following one  👀? (Same for other new added args)

https://github.com/servo/servo/blob/37259d216d7256e930fa954c2357a349933c39c3/tests/wpt/run.py#L62

Reply:
I see that the existing code has those checks, but I don't see a good reason for them.